tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -37,12 +37,12 @@ public function iSaveAnArticle()
 	}
 
 	/**
-	 * @Then I should see the :arg1 message
+	 * @Then I wait for the title :title and  see the message :message
 	 */
-	public function iShouldSeeTheMessage($message)
+	public function iShouldSeeTheMessage($title, $message)
 	{
 		$I = $this;
-		$I->waitForPageTitle('Articles');
+		$I->waitForPageTitle($title);
```

Comment:
I would suggest to do atomic steps when possible with single or minimum responsibilities.

In the present case, I see here two steps instead of one:
- wait for a title
- see a message

Why do you prefer it over, having just the responsibility of seeing a message?:

```php
    /**
     * @Then I should see the :arg1 message
     */
    public function iShouldSeeTheMessage($message)
    {
        $I = $this;
        $I->waitForText($message, 60, AdminPage::$systemMessageContainer);
        $I->see($message, AdminPage::$systemMessageContainer);
    }
```